openshift/openshift-ansible-contrib
# Make CONTAINER_ROOT_LV_SIZE configurable

#### What does this PR do?
This change makes the variable `CONTAINER_ROOT_LV_SIZE` in the docker-storage-setup file for overlayfs configurable via ansible variable `container_root_lv_size`.

#### How should this be manually tested?
Set variable `container_root_lv_size` to something else than "100%FREE" (e.g. 80%FREE) and then execute role `docker-storage-setup`. The created docker logical volume should now use the configured amount of storage (e.g. 80%).

#### Is there a relevant Issue open for this?
No.

#### Who would you like to review this?
cc: @tomassedovic PTAL
